OpenTaiko/src/Songs/CDTX.cs

```diff
@@ -1453,71 +1419,6 @@ public static string tZZ(int n) {
 			return new string(new char[] { str[n / 36], str[n % 36] });
 		}
 
-		public static void tManageKusudama(CDTX[] dtxarr) {
```

Comment:
Since every functionality is commented out and the basic if also only returns it would return in all cases, which makes this function obsolete.